docs/configuration.rst

```diff
@@ -326,18 +140,19 @@ Removes a read group set from a given data repository and dataset.
 
 .. code-block:: bash
 
-    $ ga4gh_repo remove-readgroupset path/to/datarepo aDataset aReadGroupSet
+    $ ga4gh_repo remove-readgroupset path/to/repo.db aDataset aReadGroupSet
 
 +++++++++++++++
 add-variantset
 +++++++++++++++
 
 Adds a variant set directory to a given data repository and dataset.  The
-directory should contain file(s) with extension ``.vcf.gz``. If a variant set is annotated it will be added as both a variant set and a variant annotation set.
+directory should contain file(s) with extension ``.vcf.gz``. If a variant set
+is annotated it will be added as both a variant set and a variant annotation set.
```

Comment:
We should create an issue holding the documentation requirements. This is not related to the actual changes in the PR.